### DISCUSSION TITLE
Hi how to add a portable app ?

### DISCUSSION THREAD

**Discussion by OriLiMu** [Flow-Launcher/Flow.Launcher#2486] [Q&A]
As the title said, I havent found in readme and issues.

**Comment by mblennegard** [comment]
Do you mean how to make Flow Launcher Portable if you have previously used the installer option?
If yes, you can take a look at #2507 where I had a similar question.

**Comment by taooceros** [comment]
If you refer to "find" the app in flow, you should open the setting of flow and find the progrram plugin and add the path to it.
